schema_based_on_diagram.sql

```diff
@@ -1,3 +1,53 @@
 -- make changes 
 -- add gitflow
--- add README
\ No newline at end of file
+-- add README
+
+-- create patients table :
+
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(255),
+    date_of_birth DATE
+);
+
+-- create medical_histories table and add foreign key
+
+CREATE TABLE medical_histories (
+    id INT PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INT,
+    status VARCHAR(255),
+    CONSTRAINT fk_medical_histories FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+-- create the "treatments" table with the "id" column as a foreign key referencing the "id" column in the "medical_histories" table :
+
+CREATE TABLE treatments (
+    id INT PRIMARY KEY,
+    type VARCHAR(255),
+    name VARCHAR(255),
+    CONSTRAINT fk_treatments_medical_histories FOREIGN KEY (id) REFERENCES medical_histories(id)
+);
```

Comment:
- [ ] Nice work implementing the `medical_histories` and `treatments` tables :clap: However we can notice that there is a **many-to-many** relationship between both tables. Therefore I suggest that you create the **Join table**  between those two tables.
**Example:**
```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```